src/utils.ts

```diff
@@ -326,12 +326,17 @@ export function decorateWithExplain(command: Document, explain: Explain): Docume
   return { explain: command, verbosity: explain.verbosity };
 }
 
+/**
+ * @internal
+ */
+export type TopologyProvider<T> = MongoClient | Db | Collection<T>;
+
 /**
  * A helper function to get the topology from a given provider. Throws
  * if the topology cannot be found.
```

Comment:
Let's add an `@throws` to the tsdoc here, might help the next person catch themselves if more usage of the helper is needed in the future.